src/core/thread/mle_router.hpp

```diff
@@ -489,6 +505,7 @@ class MleRouter: public Mle
     uint8_t mNetworkIdTimeout;
     uint8_t mRouterUpgradeThreshold;
     uint8_t mLeaderWeight;
+    uint32_t mLeaderPartitionId;
```

Comment:
Change this name to `mFixedLeaderPartitionId`.  Also put a comment here indicating this is only needed for certification testing.